packages/app-bridge/src/AppBridgeBlock.ts

```diff
@@ -344,9 +344,12 @@ export class AppBridgeBlock {
                   projectTypes: options.projectTypes?.map((value) => projectTypesMap[value]),
                   multiSelectionAllowed: options.multiSelection,
                   filters: [
-                      ...(options.selectedValueId !== undefined
+                      ...('selectedValueId' in options
                           ? [{ key: 'id', values: [options.selectedValueId], inverted: true }]
                           : []),
+                      ...('selectedValueIds' in options
+                          ? [{ key: 'id', values: options.selectedValueIds, inverted: true }]
```

Comment:
not sure about the uses across the application, but at some point we should deprecate selectedValueId.